docarray/base_document/document.py

```diff
@@ -34,3 +35,10 @@ def _get_field_type(cls, field: str) -> Type['BaseDocument']:
         :return:
         """
         return cls.__fields__[field].outer_type_
+
+    def __str__(self):
+        console = rich.console.Console()
```

Comment:
do we need to create a console each time or can we define it once ? (just asking)

Reply:
yes I link the later better only one console object at the module level